board/safety/safety_chrysler.h

```diff
@@ -249,35 +372,43 @@ static int chrysler_fwd_hook(int bus_num, int addr) {
   int bus_fwd = -1;
 
   // forward to camera
-  if (bus_num == 0) {
+  const bool is_buttons = (addr == chrysler_addrs->CRUISE_BUTTONS);
+  if ((bus_num == 0) && (!chrysler_longitudinal || !is_buttons)) {
```

Comment:
Why are we blocking the button message to the camera/DASM? Won't this fault it if we don't send in openpilot? Usually we let this message through in other ports.

Reply:
I don't think I tried without blocking the buttons, so it may work fine.  openpilot doesn't send any button messages for this safety mode, see openpilot/chrysler-long2 branch for how it works on that side.

Reply:
Looks like there's no need to block the button message then. We usually only do that if openpilot replaces the message we're blocking